src/supa/__init__.py

```diff
@@ -203,7 +203,8 @@ class Settings(BaseSettings):
     nsa_host: str = "localhost"
     nsa_port: str = "8080"
     nsa_name: str = "example.domain uPA"
-    nsa_provider_path: str = "/provider"
+    nsa_provider_port: str = "8443"
```

Comment:
@CsCdanNyx To let the unit test succeed, can you update line 48 of tests/documents/test_discovery.py to match this port?